src/NaturalLanguage/NaturalLanguageClient.php

```diff
@@ -218,15 +215,16 @@ public function analyzeSentiment($content, array $options = [])
      */
     public function analyzeSyntax($content, array $options = [])
     {
-        $options['features'] = ['syntax'];
-        return $this->annotateText($content, $options);
+        $syntaxResponse = $this->connection->analyzeSyntax(
+            $this->formatRequest($content, $options)
+        );
+
+        return new Annotation($syntaxResponse + ['entities' => []]);
```

Comment:
All the other methods provider an `entities` property in the response. `analyzeSyntax` [does not](https://github.com/googleapis/googleapis/pull/190/commits/879dcc2b3e8a9caae102be5d061feb210a2ab9d7#diff-3585a7188a84bb404ea0593aaaa5f5e6R858). Since `entities()` is a magic method on `Annotation` and is included in the documentation, if it isn't provided it results in a fatal error. By passing in an empty array, the method call will not raise an error, it'll just not return any values.